### ISSUE TITLE
Request: Add libNDI reverse-engineered library released under the LGPL by Jean-Baptiste Kempf

### ISSUE THREAD

**Issue by DWAMTV** [m-ab-s/media-autobuild_suite#1792]
Hi !

Like many guys, I'd like to be able to use NDI streams with FFMPEG. After the dispute with Newtek and the removal of NDI support from FFMPEG, it's really a pain in the a** to compile something that works well with newer FFMPEG updates.

As you may know, Jean-Baptiste Kempf started a new project for a LGPL libNDI library based on reverse engineering.
The project is here : https://code.videolan.org/jbk/libndi

As he says: "NDI is fun!" and not only fun, it's also very useful for more and more people in the broadcast/webcast industry.

The goal of this brand new project is to provide a way to interact with NDI streams, without requiring to use a non-open-source SDK, and targetting numerous platforms. Newtek sucked at doing it, the open source community will offer an alternative.

I believe this project could be implemented in the suite. At least, it fits the requirements for automatic install, there's no fucking  zipped download or prior install requirement and it's freely available on the videolan repo which is also the source of many projects already included in the suite.

AFAIK, libNDI is under development and does not offer all the NDI features yet (it only can receive feeds, it does not support the crappy NDI|HX sub protocol) but it seems to work and being able to receive Full NDI feeds in FFMPEG is already a great thing.

My request is simple: would you mind considering the add of this project in the suite?

Thanks and "bien à vous tous"
Guillaume

**Comment by 1480c1** [comment]
It's not yet supported in FFmpeg yet, right? I don't think I've heard about it being added to FFmpeg yet in irc

**Comment by 1480c1** [comment]
I guess https://trac.ffmpeg.org/ticket/8811 would be a good thing to track

**Comment by Streamversetv** [comment]
yes please!

**Comment by DWAMTV** [comment]
Yes I suspected it is not yet supported, but at least this FR will allow people to support the idea and justify their needs so that everybody's ready when it's possible ! ;)

**Comment by wiiaboo** [comment]
Maybe the fact that it depends on libavutil means they're thinking of importing it directly into libavformat in the future, when it's a bit less alpha-quality.

**Comment by jensstigaard** [comment]
+1

**Comment by SlipstreamCoding** [comment]
+1

**Comment by 1480c1** [comment]
Just emote 👍 on the original post
